tests/unit/test_repair.py

```diff
@@ -0,0 +1,41 @@
+import os
+import shutil
+import sys
+
+import pytest
+from auditwheel.repair import copylib
+from auditwheel.tmpdirs import InTemporaryDirectory
+from auditwheel.lddtree import lddtree
+
+@pytest.fixture
+def systemlib():
+    for libdata in lddtree(sys.executable)["libs"].values():
+        # return one of the arbitrary libraries that python is linked
+        # against, like libc.so.6. any one will do.         
+        return libdata["realpath"]
+
+
+on_supported_platform = pytest.mark.skipif(
+    sys.platform != 'linux', reason="requires Linux system"
+)
+
+@on_supported_platform
```

Comment:
Given these two tests end up invoking `patchelf` I think they belong in the integration tests.

Reply:
Okay, done. I'm not sure if the integration tests are run in an environment with patchelf installed, or if I need to move these two tests under docker as well.